### ISSUE TITLE
Feature request: Exercises view/export

### ISSUE THREAD

**Issue by eckhrd** [carpentries/workbench#18]
Hi,

it would be great if the Workbench could auto-generate a document or a website form course material that only shows exercises (with and without solutions).

As an example, we are holing an online workshop on Docker using The Carpentries Incubator [material](https://github.com/carpentries-incubator/docker-introduction). For this, I use the material as orientation during the course but show only my terminal over video conferencing. However, for the exercises, I would love to show participants a document with only the exercises and without the instructor text before and after it. Since the exercises are already written down in the material, it would be great to have this generated automatically.

**Comment by zkamvar** [comment]
One of the features we have---that I found out only works for RMarkdown material 😞---is the "handout" feature that will generate a code handout for folks to download that contained only the challenge text, no solutions. That being said, it makes a lot of sense to generate a page like this.

Are you thinking about an HTML page that contains only challenges and hints (e.g. no solutions) or are you looking for something more text-based?

**Comment by eckhrd** [comment]
I think, it would be nice to have markdown export for this, then I could easily generate a pdf or share it on a HedgeDoc document. (We have been using HedgeDoc for a couple of courses as a main hub with all course information and as a collaborative scratch pad during the course.)

It would also be great to be able to export versions with and without solutions.

**Comment by carolbooth2** [comment]
yes, this is something we would value

**Comment by cjjossart** [comment]
Yes, I agree! This would be a helpful teaching aid. Having the images and challenges possibly in the "Extract All Images" section would be helpful.  Additionally, allowing for a pdf or slides to be created as described above would be a great enhancement! Thanks!
